package.json

```diff
@@ -5,8 +5,8 @@
   "description": "Firefox Accounts Profile service.",
   "scripts": {
     "outdated": "npm outdated --depth 0",
```

Comment:
actually i think we can just drop this script and use https://github.com/mozilla/fxa-profile-server/pull/236/files#r102555395